eks-anywhere-common/Addons/Partner/dynatrace/dynatrace-variables.yaml

```diff
@@ -6,8 +6,8 @@ metadata:
   namespace: flux-system
 data:
   # api url for dynatrace environment 
-  apiurl: "<replaceme>"
+  apiurl: "https://fjy55252.sprint.apps.dynatracelabs.com/api"
```

Comment:
Is this a permanent URL? We cant have temporary URLs for this continuous testing.